src/Albany_RegressionTests.cpp

```diff
@@ -41,77 +41,71 @@ checkSolveTestResults(
   const double relTol      = testParams->get<double>("Relative Tolerance");
   const double absTol      = testParams->get<double>("Absolute Tolerance");
 
-  // Get number of responses (g) to test
-  const int numResponseTests = testParams->get<int>("Number of Comparisons");
-  if (numResponseTests > 0) {
-    ALBANY_ASSERT(
-        g != Teuchos::null,
-        "There are Response Tests but the response vector is null!");
-    ALBANY_ASSERT(
-        numResponseTests <= g->space()->dim(),
-        "Number of Response Tests (" << numResponseTests
-                                     << ") greater than number of responses ("
-                                     << g->space()->dim() << ") !");
-    Teuchos::Array<double> testValues =
-        testParams->get<Teuchos::Array<double>>("Test Values");
-
-    ALBANY_ASSERT(
-        numResponseTests == testValues.size(),
-        "Number of Response Tests (" << numResponseTests
-                                     << ") != number of Test Values ("
-                                     << testValues.size() << ") !");
-
-    Teuchos::ArrayRCP<const ST> g_view = getLocalData(g);
-    for (int i = 0; i < testValues.size(); i++) {
-      auto s = std::string("Response Test ") + std::to_string(i);
-      failures += scaledCompare(g_view[i], testValues[i], relTol, absTol, s);
-      comparisons++;
-    }
+  // Test the response (g)
+  const int numResponseTests = 1;
+  ALBANY_ASSERT(
+      g != Teuchos::null,
+      "There is Response Test but the response vector is null!");
+  ALBANY_ASSERT(
+      numResponseTests <= g->space()->dim(),
```

Comment:
I would assert that `g->space->dim()==1` explaining in the error message that responses are assumed to be scalar, and maybe remove `numResponseTests`.